src/alitra/models/translation.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+import numpy as np
+
+from alitra.models import Frame
+
+
+@dataclass
+class Translation:
+    """Translations should be expressed in the to_ frame, which are typically the asset frame"""
```

Comment:
Should the reference to `asset` be removed?

Reply:
Yes, agreed. Fixed